### ISSUE TITLE
AttributeError on chain_samples.py

### ISSUE THREAD

**Issue by SziKayLeung** [Magdoll/cDNA_Cupcake#110]
Hello Liz, 

Hope you are well. 

I am experiencing an AttributeError on the latest chain_samples.py script (version 10.0.1) - I've used both my own data and the testing data, and get the below error: 

```
python $CHAIN_SCRIPT/chain_samples.py /gpfs/mrc0/projects/Research_Project-MRC148213/sl693/softwares/Post_Isoseq3/cDNA_Cupcake/cupcake/test_data/chaining/sample.config count_fl
Sanity checking. Retrieving PBIDs from /gpfs/mrc0/projects/Research_Project-MRC148213/sl693/softwares/Post_Isoseq3/cDNA_Cupcake/cupcake/test_data/chaining/A/touse.group.txt,/gpfs/mrc0/projects/Research_Project-MRC148213/sl693/softwares/Post_Isoseq3/cDNA_Cupcake/cupcake/test_data/chaining/A/touse.gff,/gpfs/mrc0/projects/Research_Project-MRC148213/sl693/softwares/Post_Isoseq3/cDNA_Cupcake/cupcake/test_data/chaining/A/touse.count.txt...
Sanity checking. Retrieving PBIDs from /gpfs/mrc0/projects/Research_Project-MRC148213/sl693/softwares/Post_Isoseq3/cDNA_Cupcake/cupcake/test_data/chaining/B/touse.group.txt,/gpfs/mrc0/projects/Research_Project-MRC148213/sl693/softwares/Post_Isoseq3/cDNA_Cupcake/cupcake/test_data/chaining/B/touse.gff,/gpfs/mrc0/projects/Research_Project-MRC148213/sl693/softwares/Post_Isoseq3/cDNA_Cupcake/cupcake/test_data/chaining/B/touse.count.txt...
Traceback (most recent call last):
  File "/gpfs/mrc0/projects/Research_Project-MRC148213/sl693/softwares/Post_Isoseq3/cDNA_Cupcake/cupcake/tofu/counting/chain_samples.py", line 511, in <module>
    chain_samples_multithread(sample_dirs, sample_names, group_filename, gff_filename, count_filename, args.field_to_use, args.fuzzy_junction, args.allow_5merge, args.max_3_diff, fastq_filename, cpus=args.cpus)
  File "/gpfs/mrc0/projects/Research_Project-MRC148213/sl693/softwares/Post_Isoseq3/cDNA_Cupcake/cupcake/tofu/counting/chain_samples.py", line 432, in chain_samples_multithread
    addon_fq=addon_fq)
  File "/gpfs/mrc0/projects/Research_Project-MRC148213/sl693/softwares/Post_Isoseq3/cDNA_Cupcake/cupcake/tofu/counting/chain_samples.py", line 242, in combine_split_chained_results
    rec_list.append(sp.MatchRecord(ref_id=ref_id, addon_id=addon_id, rec=r2,
AttributeError: module 'cupcake.tofu.counting.combine_abundance_across_samples' has no attribute 'MatchRecord'


```
I am wondering whether it's something that I'm doing wrong? 

Thank you, 
Szi Kay

**Comment by anilkanthi** [comment]
Hi @SziKayLeung 

I am getting the same error. How were you able to resolve this one?

Traceback (most recent call last):
  File "/gpfs/share/apps/anaconda3/cpu/5.3.1/bin/chain_samples.py", line 4, in <module>
    __import__('pkg_resources').run_script('cupcake==10.0.0', 'chain_samples.py')
  File "/gpfs/share/apps/anaconda3/cpu/5.3.1/lib/python3.7/site-packages/pkg_resources/__init__.py", line 661, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/gpfs/share/apps/anaconda3/cpu/5.3.1/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1441, in run_script
    exec(code, namespace, namespace)
  File "/gpfs/share/apps/anaconda3/cpu/5.3.1/lib/python3.7/site-packages/cupcake-10.0.0-py3.7-linux-x86_64.egg/EGG-INFO/scripts/chain_samples.py", line 511, in <module>
    chain_samples_multithread(sample_dirs, sample_names, group_filename, gff_filename, count_filename, args.field_to_use, args.fuzzy_junction, args.allow_5merge, args.max_3_diff, fastq_filename, cpus=args.cpus)
  File "/gpfs/share/apps/anaconda3/cpu/5.3.1/lib/python3.7/site-packages/cupcake-10.0.0-py3.7-linux-x86_64.egg/EGG-INFO/scripts/chain_samples.py", line 432, in chain_samples_multithread
    addon_fq=addon_fq)
  File "/gpfs/share/apps/anaconda3/cpu/5.3.1/lib/python3.7/site-packages/cupcake-10.0.0-py3.7-linux-x86_64.egg/EGG-INFO/scripts/chain_samples.py", line 230, in combine_split_chained_results
    if len(matches) == 0: rec_list.append(sp.MatchRecord(ref_id=ref_id, addon_id='NA', rec=d_ref[ref_id],
AttributeError: module 'cupcake.tofu.counting.combine_abundance_across_samples' has no attribute 'MatchRecord'

**Comment by Magdoll** [comment]
Hi @anilkanthi ,
Cupcake is now at v11.0.0. Can you please first update to the latest version and try again? 
If issue persists, let me know.
Thanks,
-Liz

**Comment by anilkanthi** [comment]
Hi @Magdoll 
It still shows the error in the latest version.

**Comment by anilkanthi** [comment]
Hi @Magdoll 
The error is resolved once updated to v11.0.0
Thank you!
